src/js/pages/TwitterHandleLanding.jsx

```diff
@@ -52,36 +52,19 @@ export default class TwitterHandleLanding extends Component {
   }
 
   // eslint-disable-next-line camelcase,react/sort-comp
-  UNSAFE_componentWillReceiveProps (nextProps) {
-    // console.log('TwitterHandleLanding componentWillReceiveProps');
-    let activeRoute = '';
-    let nextTwitterHandle;
-    // eslint-disable-next-line react/prop-types
-    if (nextProps.match && nextProps.location) {
-      // eslint-disable-next-line react/prop-types,camelcase
-      const { location: { pathname }, match: {params: { twitter_handle} }  } = nextProps;
-      activeRoute = pathname;
-      nextTwitterHandle = twitter_handle;
-      // eslint-disable-next-line react/prop-types
-    } else if (this.props && this.props.location) {
-      // eslint-disable-next-line react/prop-types
-      const { location: { pathname } } = this.props;
-      activeRoute = pathname;
+  componentDidUpdate(prevProps) {
+    const { match: { params: { twitter_handle: twitterHandle } }, location: { pathname: activeRoute } } = this.props;
+    // if-statements prevents infinite rendering
+    if (prevProps.location.pathname !== this.props.location.pathname) {
```

Comment:
I seem to remember having a problem with some situations where .location wasn't present, so trying to access an element of an empty variable throws an error. I recommend creating a "prevPathname" and "currentPathname" variable that will exist whether or not "prevProps.location" or "this.props.location" exist, before you do this comparison:
if (prevProps.location.pathname !== this.props.location.pathname)

This same comment applies to the next comparison:
if (prevProps.match.params.twitter_handle !== this.props.match.params.twitter_handle)